configs/hoodi/testnet.seed.yaml

```diff
@@ -0,0 +1,34 @@
+# yaml-language-server: $schema=./../../schemas/seed-schema.json
+deployed:
+  l1:
+    - &OssifiableProxy1 "0xbE861866E2Ca8f401e2b51b2cFb36A61B0Bf6840"
+    - &AppProxyUpgradeable "0x04d160820C0f2E2C693D9Eb26078189D10A1a3e1"
+    - &OssifiableProxy2 "0x7d7EcCE45cAc6eF043f34e8049399b4b03044F97"
```

Comment:
```suggestion
    - &Accounting "0x7d7EcCE45cAc6eF043f34e8049399b4b03044F97"
```

Reply:
And so on... here we do not need exact contracts, but real meaning of the address more, to make the YAML readable